db/migrate/20210406060434_fix_topic_user_bookmarked_sync_issues_again.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class FixTopicUserBookmarkedSyncIssuesAgain < ActiveRecord::Migration[6.0]
+  def up
```

Comment:
disable ddl transaction here. This can fail halfway and then recover cleanly. Means we hold locks for a lot less time.